docs/en/stack/data-frames/dataframes.asciidoc

```diff
@@ -36,9 +36,15 @@ more about the supported aggregations and group-by fields, see
 As an optional step, it's also possible to add a query to further limit the 
 scope of the aggregation.
 
-IMPORTANT: In 7.2, you can build {dataframes} on the top of a static indices. 
-When new data comes into the index, you have to perform the transformation again 
-on the altered data.
+The {dataframe-transform} performs a composite aggregation that 
+paginates through all the data defined by the source index query. The output of
+the aggregation is stored in a destination index, known as a {dataframe}. The 
+processing of a single composite aggregation that the {dataframe-transform} 
```

Comment:
Maybe this could be re-written along these lines?:

Each time the {dataframe-transform} queries the source index, it creates a _checkpoint_.